db/migrate/20191023232724_add_description_photo_status_to_products.rb

```diff
@@ -0,0 +1,7 @@
+class AddDescriptionPhotoStatusToProducts < ActiveRecord::Migration[5.2]
+  def chang
```

Comment:
This should be def change
```suggestion
  def change
```